### ISSUE TITLE
dart-services no longer works with latest dev SDK

### ISSUE THREAD

**Issue by jcollins-g** [dart-lang/dart-services#343]
Dependency problems with shelf_cors vs. shelf seem to be the lynchpin of a large number of pub errors.

https://github.com/gmosx/dart-shelf_cors/pull/5

I also want to use this issue to track making travis update the SDK version on its own to match dev, so that tests catch these failures faster.

**Comment by jcollins-g** [comment]
Another issue has come up; the rpc package no longer works on the latest dev SDK either.  dart-lang/rpc#123.  Without this we can't regenerate our interface properly.

**Comment by jcollins-g** [comment]
RPC fixes are getting close to landing; hoping to unblock this this week.
